### PR TITLE
US259 add feature file and step definitions - Do not merge till code is ready

### PR DESCRIPTION
**Pull request by Pheonnexx**

### CODE REVIEW

File: features/provide_vat_receipt.feature [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+Acceptance criteria:
+
+1) Must contain:
+    - Unique invoice number that follows on from the last invoice
+    - LR business name and address
+    - LR VAT number
+    - The tax point (or ‘time of supply’) if this is different from the invoice date [see activity point below for more info on this point]
+    - Description of the goods or services
+    - Rate of VAT charged per item - if an item is exempt or zero-rated make clear no VAT on these items
```
Comment (RichardProsser) [inline features/provide_vat_receipt.feature:15]: Strictly speaking, VAT **is** charged in the "zero-rated" case (I believe) but is zero!

File: features/provide_vat_receipt.feature [hunk 2]
```diff
@@ -0,0 +1,33 @@
+@US259
+Feature: Add VAT receipt at the bottom of the printed summary page
```
Comment (RichardProsser) [inline features/provide_vat_receipt.feature:2]: I thought that the VAT receipt was meant to be separate. Is that no longer true?

File: features/provide_vat_receipt.feature (hunk 1 above)
Comment (woodingd) [inline features/provide_vat_receipt.feature:15]: For a fee of £0, the VAT receipt shows VAT as £0 as well.

File: features/provide_vat_receipt.feature (hunk 2 above)
Comment (woodingd) [inline features/provide_vat_receipt.feature:2]: I think as a 'nice to have' in the future we will want to have the VAT receipt printable separately but as an MVP having the VAT receipt on the summary and on the printout is acceptable.

**Pull request merged**